portfolio/src/main/webapp/index.html

```diff
@@ -51,6 +51,21 @@ <h3>
           Click <a href="pictures.html">here</a> to check out a gallery of my pictures. See my hiking trips,
           dogs, and favorite places!
       </p>
+
+      <!--picture upload form-->
+        <form action="/upload" method="POST" enctype="multipart/form-data">
+            <p>Upload one of your own favorite pictures!</p>
+            <input class = "inputI" type="file" name="image">
```

Comment:
What does inputI and submitI stand for?

Reply:
This is the inputted image and submit button for the image. I changed the names the inputImage and submitImage.